docs/source/federation-version-support.mdx

```diff
@@ -27,15 +27,23 @@ The table below shows which version of federation each router release is compile
     <tbody>
     <tr>
         <td>
-            v1.29.0 and later (<a href="https://github.com/apollographql/router/releases">see latest releases</a>)
+            v1.29.1 and later (<a href="https://github.com/apollographql/router/releases">see latest releases</a>)
         </td>
         <td>
             2.5.3
         </td>
     </tr>
     <tr>
         <td>
-            v1.28.0 - v1.28.1
+            ⚠️ v1.29.0
+        </td>
+        <td>
+            2.5.3
```

Comment:
2.5.3 is listed in the row above is it intentional to have two rows with the same value?.

Reply:
Yes, the version with the ⚠️ is specific.

Reply:
v1.29.1 with 2.5.3 is great.  1.29.0 with 2.5.3 is not.